tests/texture/stargz/stargz.index-lower.json

```diff
@@ -1,15 +1,6 @@
 {
 	"version": 1,
 	"entries": [
-		{
-			"name": "",
-			"type": "dir",
-			"modtime": "2020-09-10T07:40:29Z",
-			"mode": 493,
-			"userName": "root",
-			"groupName": "root",
-			"NumLink": 0
-		},
```

Comment:
Why removing this directory entry?

Reply:
The estargz image format has changed a lot, for a real estargz image, the TOC file in each layer does not contain root inode, so we need to prepend one automatically before dumping bootstrap.